src/classes/UTIL_Namespace.cls

```diff
@@ -146,4 +146,16 @@ public class UTIL_Namespace {
         }
         return email;
     }
+
+    /*******************************************************************************************************
+    * @description Only remove npsp prefix, don't attempt to remove others
+    * 
+    * @param str token name 
+    * @return token name, with namespace prefix removed, if present.
+    ********************************************************************************************************/
+    public static string StrTokenRemoveNSPrefixNpspOnly(string str) {
+        if(str.startsWith('npsp__'))
```

Comment:
Include the {}s on the if statement

Reply:
Also for method naming conventions we would prefer to be compliant with current standards, even if it's at odds with the rest of the class style - so "removeNSPrefixNPSPOnly()" or whatever name makes sense that isn't hungarian notation